wsmaster/che-core-sql-schema/src/main/resources/che-schema/6.4.0/2__add_k8s_metadata.sql

```diff
@@ -0,0 +1,75 @@
+--
+-- Copyright (c) 2012-2018 Red Hat, Inc.
+-- All rights reserved. This program and the accompanying materials
+-- are made available under the terms of the Eclipse Public License v1.0
+-- which accompanies this distribution, and is available at
+-- http://www.eclipse.org/legal/epl-v10.html
+--
+-- Contributors:
+--   Red Hat, Inc. - initial API and implementation
+--
+
+-- Runtime ---------------------------------------------------------------------
+CREATE TABLE che_k8s_runtime (
+    workspace_id    VARCHAR(255)    NOT NULL,
```

Comment:
workspace_id uniuque

Reply:
It is a primary key. Are you sure that I must specify unique here? I checked other places like workspace, stack tables, id field is not marked as unique.

Reply:
ok